packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -429,14 +429,23 @@ export function _SignInStart(): JSX.Element {
       <Card.Root>
         <Card.Content>
           <Header.Root showLogo>
-            {isCombinedFlow ? (
-              <Header.Title localizationKey={localizationKeys('signIn.start.titleCombined')} />
-            ) : (
-              <>
-                <Header.Title localizationKey={localizationKeys('signIn.start.title')} />
-                <Header.Subtitle localizationKey={localizationKeys('signIn.start.subtitle')} />
-              </>
-            )}
+            <Header.Title
+              localizationKey={
+                isCombinedFlow ? localizationKeys('signIn.start.titleCombined') : localizationKeys('signIn.start.title')
+              }
+            />
+            <Header.Subtitle
+              localizationKey={
+                isCombinedFlow
+                  ? localizationKeys('signIn.start.subtitleCombined')
+                  : localizationKeys('signIn.start.subtitle')
+              }
+              sx={{
+                '&:empty': {
+                  display: 'none',
+                },
+              }}
```

Comment:
By default the sign in start subtitle is undefined, and we use the empty selector to ensure the element does not take up any space within the dom. This is to enable folks to see that there is a localization option available for subtitle when viewing source in the dom. Then they can add the localization and it will show for them.